npm/test/db/db.test.ts

```diff
@@ -201,15 +212,32 @@ tap.teardown(async () => {
 });
 
 tap.test('dbs', async () => {
+  // We need this to ensure that the test runs atleast once.
+  // It is quite easy to skip the test by mistake in the future
+  // if one of the conditions change and it goes unnoticed.
+  let has_non_default_postgres_schema_test_ran = false;
   for (const idx in connectionStores) {
     const connectionStore = connectionStores[idx];
     const ttlStore = ttlStores[idx];
     const dbEngine = dbs[idx].engine!;
-    let dbType = dbEngine;
+    let dbType = dbEngine.toString();
     if (dbs[idx].type) {
       dbType += ': ' + dbs[idx].type;
     }
 
+    tap.test('Test non default postgres schema', (t) => {
+      if (dbType === 'sql: postgres' && dbs[idx].postgres?.schema === non_default_schema) {
+        t.same(
+          connectionStore['db']['db']['dataSource']['createQueryBuilder']()['connection']['options'][
+            'schema'
```

Comment:
Not the cleanest solution, but since we want to expose a limited number of methods in DatabaseDriver, we needed to access private members of the class to test our changes